rust/vectordb/src/table.rs

```diff
@@ -214,32 +264,44 @@ pub trait Table: std::fmt::Display + Send + Sync {
     ///
     /// ```no_run
     /// # use std::sync::Arc;
-    /// # use vectordb::connection::{Database, Connection};
```

Comment:
Don't have to do this in this PR, but: If we feel like merge_insert is sufficiently tested, we could use [the technique described here to shorten this and other examples](https://rust-unofficial.github.io/patterns/idioms/rustdoc-init.html).

Reply:
Oh man, I had been looking for something like this a while ago.  This is a very cool idea.  I will wait for a follow-up though.  I want to revisit the examples in a little more detail once the refactor is done (and I need to update the rst).